src/components/__tests__/codeBlock.test.js

```diff
@@ -1,13 +1,35 @@
 import React from 'react';
 import {create} from 'react-test-renderer';
 
+import { CodeBlock } from '../codeBlock';
 import {_setCachedCodeKeywords, CodeContextProvider, DEFAULTS} from '../codeContext';
 import {makeKeywordsClickable} from '../codeKeywords';
 
 function CodeWrapper({children, ...props}) {
   return <code {...props}>{children ? makeKeywordsClickable(children) : children}</code>;
 }
 
+// Tests that CodeBlock renders SignInNote when necessary.
```

Comment:
Hey @natikgadzhi ! 

I think you can add a simple jest mock for it like this:

```suggestion
jest.mock('../signInNote', () => ({
  SignInNote: () => {
    return <mock-signInNote />;
  },
}));

// Tests that CodeBlock renders SignInNote when necessary.
```

Reply:
I'm on it, one sec!